src/main/java/org/sasanlabs/service/vulnerability/openRedirect/Http3xxStatusCodeBasedInjection.java

```diff
@@ -53,7 +53,7 @@ public class Http3xxStatusCodeBasedInjection {
 
     private static final String LOCATION_HEADER_KEY = "Location";
     private static final String RETURN_TO = "returnTo";
-    private static final Set<String> WHITELISTED_URLS = new HashSet<>(Arrays.asList("/"));
+    private static final Set<String> WHITELISTED_URLS = new HashSet<>(Arrays.asList("/", "/VulnerableApp/"));
```

Comment:
if possible, can you please write a unit test case?

Reply:
Sure thing!